.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, Impact/Severity: Low, Priority: Low, waiting for collaborator
+  review'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
can also be described by why they want the feature

Reply:
Remove this one?